lib/repositories/challenge_comment_repo.rb

```diff
@@ -6,8 +6,7 @@ def create(attrs)
   end
 
   def recent
-    challenge_comments.recent.ordered_by_date.limit(20).to_a
-    # challenge_comments.ordered_by_date.limit(20).to_a
+    challenge_comments.recent.ordered_by_created_at.limit(20).to_a
```

Comment:
I think we should move ordering into the `recent` method. (My initial implementation looks underdone). `ordered_by_date` can be removed if it's not used.